### ISSUE TITLE
unable to get mic to work...

### ISSUE THREAD

**Issue by sdetweil** [mykle1/MMM-VoiceControlMe#2]
use same config as MMM-voice, micrrophone: "1,0"

get error (debug: true)

Error opening audio device plughw:1,0 for capture: Device or resource busy
FATAL: "continuous.c", line 245: Failed to open audio device

MMM-voice works ok..

**Comment by TheStigh** [comment]
Hi @sdetweil seems you have something else occupying the mic already.
Try to disable MMM-voice and test.

**Comment by sdetweil** [comment]
No, the device name changed. hw:1,0 and default work, plughw:1,0 doesn't. Open PR with fix on pocketsphinx_continous. He is examining. plugwh: is hard coded in psc 

Something in alsa changed

**Comment by TheStigh** [comment]
I have three devices, one where mic is on device 0, one on device 1 and one on device 2. All of them work. Can you please explain better what you believe the issue is ?

You should use only (if microphone is on device 1): 

`microphone: 1`

**Comment by sdetweil** [comment]
I have been using this mic device plughw:1,0 for over two years on both smart-mirror and MagicMirror.

I installed MMM-Hotword, and suddenly plughw:1,0 will not work. I debugged it all the way to the alsa open, which returned unknown device.  I changed to default and it worked fine.  MMM-voice was also impacted
